cpp_format_tools/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG ALPINE_VERSION=3.13
+
+FROM alpine:${ALPINE_VERSION}
+LABEL maintainer="The OpenTelemetry Authors"
+RUN apk update
+RUN apk add --no-cache clang=10.0.1-r0 python3 py3-pip git curl
```

Comment:
Move all the version definitions to the beginning of this file, like `ALPINE_VERSION`?

Reply:
Done.